src/Module.Server/ModuleData/crafting_templates.xml

```diff
@@ -9260,4 +9260,73 @@
       <UsablePiece piece_id="crpg_meowhammer_handle_h3" />
     </UsablePieces>
   </CraftingTemplate>
+  <CraftingTemplate id="Wakizashi" item_type="OneHandedWeapon" modifier_group="sword" item_holsters="katana_left_hip:katana_left_hip_2:katana_left_hip_3" piece_type_to_scale_holster_with="Blade" hidden_piece_types_on_holster="Blade" default_item_holster_position_offset="0,0,-0.1">

```

Comment:
Indeed this must be done in item balancing , and not here

Reply:
They did not exist before because we did not have the custom holster positions.
Now that we want make use of them, those custom templates need to be added so weapons can use them.
Both need to happen on the other repo since the exporter will crash if the custom item_holsters.xml is not available but weapons are using the new crafting template that references those holster positions.

My suggestion would be only adding the custom item_holsters.xml here, syncing it to the item balancing repo and then adding crafting templates and the weapons that use them on there.
That way we have no situation on any repo would cause issues.